.travis.yml

```diff
@@ -15,60 +15,90 @@ cache:
 
 env:
   global:
-    - SERVICE_MANAGER_VERSION="^3.0.3"
-    - SITE_URL: https://zendframework.github.io/zend-paginator
-    - GH_USER_NAME: "Matthew Weier O'Phinney"
-    - GH_USER_EMAIL: matthew@weierophinney.net
-    - GH_REF: github.com/zendframework/zend-paginator.git
-    - secure: "ikKFx5tciHd333+GjlprvgnrBjWo1hpzR7W/MHvi+MZHaipIaG8BqGudgEhgjQryMeJnk8d/ta+lia6OGwKYdsy575/iJAFns5y1tOlPXT+paJ98vdiMJmSbRfC5mQ32SiCsnp780Khhbhrmx14WHza2aBxFFewQ8oYjfQuEZY7LKCLzQkpnsMTNNDsXWVOlYuNxjIZ89eahUDZPp/p+gLHZvC0JcNi/LXXicNo8qcjsOQTVvXcWQJn1ghBAVk8WEEoRpuQ+dAW6ULhInpNEUSuskWYcXBOGfPP42Bb37hIiRERkxJWqdLwSZihh5Hne+NXLkJBLE7QKA0OJxsVUQmX4/ggn8HW3CVV+7LLeBW89NU0grNszwmARUWgkEUpEF7YXq1vHi8P4EJJ1kSXcSNEynVYM1ZIKx2qOKBor2BFS49+iwgQgFYwuHXiC3Oetg+vHWcJm/pyRq+xhqukoJ92S9Pwu3cNbyttEM/NRdfrT8p7tBaYSck+HpJgNpgmUyBMuI7xz+eMgIU/ZJfiWPZRr9Oom13+z3FW8dd+jkpMjpBT5TnGNLU4DRz4H7WoNLVR9wdPdTDOD85qDv1qivrr2GKRiKlH3ySaHPnWu98Eu9YdnvwQn0YW7HMO20OgdNa7EOz4zvyyX4b4tBRyPk9GI/bhoa0VQJWJhLE3Cthk="
+    - COMPOSER_ARGS="--no-interaction"
+    - COVERAGE_DEPS="satooshi/php-coveralls"
+    - LEGACY_DEPS="phpunit/phpunit"
 
 matrix:
   include:
     - php: 5.5
       env:
-        - CS_CHECK=true
+        - DEPS=lowest
+    - php: 5.5
+      env:
+        - deps=locked
     - php: 5.5
       env:
-        - SERVICE_MANAGER_VERSION="^2.7.5"
+        - deps=latest
     - php: 5.6
       env:
+        - DEPS=lowest
+    - php: 5.6
+      env:
+        - deps=locked
         - TEST_COVERAGE=true
-        - DEPLOY_DOCS="$(if [[ $TRAVIS_BRANCH == 'master' && $TRAVIS_PULL_REQUEST == 'false' ]]; then echo -n 'true' ; else echo -n 'false' ; fi)"
-        - PATH="$HOME/.local/bin:$PATH"
     - php: 5.6
       env:
-        - SERVICE_MANAGER_VERSION="^2.7.5"
+        - deps=latest
     - php: 7
+      env:
+        - DEPS=lowest
     - php: 7
       env:
-        - SERVICE_MANAGER_VERSION="^2.7.5"
-    - php: hhvm 
-    - php: hhvm 
+        - deps=locked
+        - CS_CHECK=true
+    - php: 7
+      env:
+        - deps=latest
+    - php: 7.1
+      env:
+        - DEPS=lowest
+    - php: 7.1
+      env:
+        - deps=locked
+    - php: 7.1
       env:
-        - SERVICE_MANAGER_VERSION="^2.7.5"
+        - deps=latest
+    - php: 7.2
+      env:
+        - DEPS=lowest
+    - php: 7.2
+      env:
+        - deps=locked
+    - php: 7.2
+      env:
+        - deps=latest
+    - php: hhvm
```

Comment:
We dropped HHVM in many other components, why not here?

Reply:
It is dropped, as of 5dcb0c8; not sure why the diff doesn't show it.

Reply:
You are right. Travis shows the correct result!